exporter/prometheusremotewriteexporter/README.md

```diff
@@ -51,4 +51,4 @@ Several helper files are leveraged to provide additional capabilities automatica
 
 - [HTTP settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/confighttp/README.md)
 - [TLS and mTLS settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md)
-- [Queuing, retry and timeout settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md)
+- [Retry and timeout settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md), not that the exporter doesn't support `sending_queue`.
```

Comment:
This is a double negative, could we perhaps make it:

    The exporter does not support sending_queue

Reply:
Sorry, it was a typo. I wanted to say "note that...". Fixed now.